.generator/templates/group_test.go

```diff
@@ -0,0 +1,262 @@
+package okta
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func cleanUpGroupRule(groupRuleId string) (err error) {
+	_, err = apiClient.GroupApi.DeleteGroupRule(apiClient.cfg.Context, groupRuleId).Execute()
+	return
+}
+
+func Test_Get_Group(t *testing.T) {
+	t.Parallel()
+	group, _, err := setupGroup(randomTestString())
+	require.NoError(t, err, "Creating a new group should not error")
+	t.Run("get group by id", func(t *testing.T) {
+		gid, _, err := apiClient.GroupApi.GetGroup(apiClient.cfg.Context, group.GetId()).Execute()
+		require.NoError(t, err, "Could not get group by ID")
+		assert.Equal(t, group.GetId(), gid.GetId())
+	})
+	err = cleanUpGroup(group.GetId())
+	require.NoError(t, err, "Clean up group should not error")
+}
+
+func Test_Get_List_Group(t *testing.T) {
+	t.Parallel()
+	group, _, err := setupGroup(randomTestString())
+	require.NoError(t, err, "Creating a new group should not error")
+	t.Run("get list group", func(t *testing.T) {
+		gs, _, err := apiClient.GroupApi.ListGroups(apiClient.cfg.Context).Execute()
+		require.NoError(t, err, "Could not get list group")
+		var createdGroupInList bool
+		for _, g := range gs {
+			if group.GetId() == g.GetId() {
+				createdGroupInList = true
+			}
+		}
+		assert.True(t, createdGroupInList, "Could not find group from list")
+	})
+	err = cleanUpGroup(group.GetId())
+	require.NoError(t, err, "Clean up group should not error")
+}
+
+func Test_Search_Group(t *testing.T) {
+	t.Parallel()
+	groupName := randomTestString()
+	group, _, err := setupGroup(groupName)
+	require.NoError(t, err, "Creating a new group should not error")
+	t.Run("get list group", func(t *testing.T) {
```

Comment:
Some of the test runs didn't get updated with their name from "get list group" being used as a reference.
Lines 52, 56, 74